src/test/java/com/VetApp/PurrgentCare/services/AccountServiceImplementationTest.java

```diff
@@ -208,4 +231,71 @@ public void toggleAccount_whenAccountNotExists_throwEntityNotFoundException() {
         then(exception.getMessage()).contains(String.valueOf(accountId));
 
     }
+
+    // Test associatePeople when account exists
+        //This method adds a list of personIds to an account
+        //
+    @Test
+    public void associatePeople_whenAccountExists_returnAssociatedPeople() {
+
+        // Variables used everywhere
+        final var accountId = new Random().nextInt(1000);
+        final var personId = new Random().nextInt(1000);
+        final var active = Boolean.TRUE;
+        final var dateCreated = new Date();
+
+        // Build request object
+        final var peopleIdList = new ArrayList<Integer>(personId);
+        final var request = new AssociatePeopleWithAccountRequest();
+        request.accountId = accountId;
+        request.personIds = peopleIdList;
+
+        // Build person object
+        final var person1 = Person.builder()
+                .id(personId)
+                .build();
+
+        // Build pet object
+        final var pet1 = Pet.builder()
+                .build();
+
+        // Build original account object
+        final var originalAccount = Account.builder()
```

Comment:
How did you get the builders to work? Jesse and I sort of tried this and weren't able to get it going/accepting the .builder annotation for some different Classes

Reply:
The builder methods only works for classes that have the @Builder annotation in their class.
For example, looking at Account, Pet, and Person classes... we have the @Builder annotation.
For the request and response classes, we do not.

Reply:
Okay, I think we tried to do it for the req/response classes and that's where we went wrong.

Reply:
Yupp, makes sense. We can add the Builder annotation to make that an option if we want.
It's at our discretion.